### ISSUE TITLE
GoFiber Middleware

### ISSUE THREAD

**Issue by gaby** [eko/authz#385]
Hello @eko I was wondering if you have thought about making a contrib GoFiber middleware for `authz`?

**Comment by eko** [comment]
Hi @gaby,

For sure it will be interesting to have one :)

If you want to work on it do not hesitate, elsewhere I will look to implement it as soon as possible.

Maybe we could push it on https://github.com/gofiber/contrib?

**Comment by gaby** [comment]
@eko Agree, contrib repo would be the perfect place. If you can come up with a proof of concept we can help develop it further into a production middleware. 

Thanks!
